frontend/src/pages/Signup.tsx

```diff
@@ -191,15 +207,36 @@ export default function Signup() {
               ) : (
                 <div>
                   {chain?.open_to_new_members && chain.published ? (
```

Comment:
Hi @lil5, I think this condition could cause an issue when the user tries to register without being associated with a loop. Maybe we could change it to something like this, what do you think?
``` 
 {!chain || chain?.open_to_new_members && chain.published ? 
```